### ISSUE TITLE
count_ss and count_s Not included XH and SH

### ISSUE THREAD

**Issue by Arsenolite** [ppy/osu-api#204]
![image](https://user-images.githubusercontent.com/26121316/34453047-ef33bb56-ed85-11e7-98ba-ffc25ced9c82.png)
![image](https://user-images.githubusercontent.com/26121316/34453048-fae9b37e-ed85-11e7-9cca-5e4b14675fbe.png)

add count_rank_sh and count_rank_ssh pls.

**Comment by tetsuo286** [comment]
I could be wrong, but didn't `count_rank_ss` and `count_rank_s` previously include the hidden ranks in the value? Looks like that's not the case at the moment.

**Comment by Arsenolite** [comment]
@tetsuo286 yes, `count_rank_ss` and `count_rank_s` previously include the hidden ranks in the value. And now it looks like ppy has added the `ssh/sh` field; the value of `count_rank_ssh` works fine, but the value of `count_rank_sh` is simply equals the value of `count_rank_s` , instead of the the correct value.
